tslearn/metrics/cycc.py

```diff
@@ -0,0 +1,88 @@
+STUFF_cycc = "cycc"
+
+import numpy as np
+from numba import njit, prange
+
+__author__ = "Romain Tavenard romain.tavenard[at]univ-rennes2.fr"
+
+DTYPE = float
```

Comment:
I think you may want to remove `DTYPE`, and clearly shows the word "float" in the code below. Because, it is not easy to go through the code and then see `DTYPE`. Then, you need to go back and check what `DTYPE` was. I mean...this may not be necessarily cleaner.

Reply:
Ok, thank you.